test/extensions/firstorder/firstorder_settings.py

```diff
@@ -81,14 +81,27 @@
     {
         "input_fn": lambda: torch.rand(3, 3, 7),
         "module_fn": lambda: torch.nn.Sequential(
-            torch.nn.Conv1d(3, 2, 2),
+            torch.nn.Conv1d(3, 2, 2, bias=False),
```

Comment:
Instead of modifying an existing test, could you add a new one with `bias=False`?